### DISCUSSION TITLE
How to install TALLY package.

### DISCUSSION THREAD

**Discussion by WU2JING** [deepmodeling/deepmd-kit#2429] [General]
Now, I try to use the TALLY package with deep potential. But, the lammps do not include TALLY package of off-line packages.
To use the TALLY package, whether I need to recompile the deepmd-kit from source code.
Thank you.

**Comment by njzjz** [comment]
You can rebuild the lammps package following https://docs.deepmodeling.com/projects/deepmd/en/master/install/build-conda.html
